### PR TITLE
Rokt Audience Set to Publicly Available

### PR DESCRIPTION
**Pull request by GoodmanBen**
### Proposed changes
Set the Rokt Audiences Destination docs to public.

### Merge timing
- Good to merge once https://github.com/segmentio/action-destinations/pull/2606 is merged in and has been tested in private beta 

### Related issues
N/A

### CODE REVIEW

**Comment by stayseesong** [comment]
@GoodmanBen is this meant to be kept in draft mode?

**Comment by GoodmanBen** [comment]
Hi @stayseesong thank you for checking. Yes the corresponding changes have been merged in for the linked PR, but not yet deployed due to a deploy freeze around Thanksgiving. It sounds like the changes will be deployed on the 3rd of December, at which point we will test in private beta and then release this to public beta. Will ping here once that is complete and we can move this PR out of draft mode.

**Comment by GoodmanBen** [comment]
Hi @stayseesong, this has now been tested and validated in our private beta deployment and is ready for review.

File: src/connections/destinations/catalog/actions-rokt-audiences/index.md [hunk 1]
```diff
@@ -43,25 +43,24 @@ To add the Rokt Audiences (Actions) destination:
...
 
 9. Go to the **Settings** tab and select the toggle to **Enable** the destination.
 
 10. Select your space, and navigate to **Engage > Audiences**. Select the source audience that you want to send to your Rokt Audiences (Actions) destination.
 
-11. Click **Add Destinations** and select the Rokt Audience (Actions) destination you created. In the settings that appear on the right-hand side, toggle the **Send Track** option on and **Send Identify**. Click **Save**. 
+11. Click **Add Destinations** and select the Rokt Audience (Actions) destination you created.
+In the settings that appear on the right-hand side, toggle the **Send Track** option on and **Send Identify**.
+
+12. Within the same panel view, select "Default Setup". You may now hit  Click **Save** in the top right corner.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-rokt-audiences/index.md:61]:
```suggestion
11. Click **Add Destinations** and select the Rokt Audience (Actions) destination you created.
12. In the settings that appear on the right-hand side, toggle on the **Send Track** and **Send Identify** option.
13. Select **Default Setup**. 
14. Click **Save** in the top right corner.
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

**Comment by tcgilbert** [comment]
@stayseesong can we get this included in tomorrow's deployment?

**Review by stayseesong — APPROVED** [review]

**Pull request merged**